frontend/src/utils/polls/videos.ts

```diff
@@ -1,16 +1,37 @@
 import { TFunction } from 'react-i18next';
-import { EntitiesService, Entity, TypeEnum } from 'src/services/openapi';
+import { PollsService, Recommendation } from 'src/services/openapi';
 import { OrderedDialogs } from 'src/utils/types';
 
-let VIDEOS: Promise<Entity[]> | null = null;
+let VIDEOS: Promise<Recommendation[]> | null = null;
 
-export function getTutorialVideos(): Promise<Entity[]> {
+/**
+ * Return a list of `Recommendation` for the tutorial.
+ *
+ * The recommendations returned:
+ *   - are safe recommendations
+ *   - are in the TOP 40 of all-time recommendations
+ *   - have a maximum duration of 5 minutes
+ *
+ * TODO:
+ *   - make the recommendations' language match the user language
+ *   - if no reco. are available in this language, use `en`?
+ */
+export function getTutorialVideos(): Promise<Recommendation[]> {
   if (VIDEOS != null) {
     return VIDEOS;
   }
-  VIDEOS = EntitiesService.entitiesList({
-    type: TypeEnum.VIDEO,
-    limit: 8,
+
+  const minutesMax = 5;
+  const top = 40;
```

Comment:
Isn't 40 a bit restrictive? I fear that excessively promoting a small number of videos which would receive a lot of comparisons would possibly reinforce their ranking among the top recommendations. 

In the longer term a filter on the tournesol score may be more relevant, but in the meantime why not setting a higher limit? 100 ?

Reply:
You're right, 100 will be better to avoid promoting the same content.

Reply:
As I don't have my computer with me, feel free to update this value.